tests/check_elemwise.c

```diff
@@ -479,6 +479,12 @@ START_TEST(test_basic_broadcast) {
 
   ck_assert_int_eq(GpuElemwise_call(ge, rargs, GE_NOCOLLAPSE), GA_VALUE_ERROR);
 
+  ck_assert_int_eq(GpuElemwise_call(ge, rargs, GE_NOCOLLAPSE | GE_BROADCAST), GA_VALUE_ERROR);
```

Comment:
This is a poor test.  It would pass even if the new broadcast check doesn't do anything, since we try to output (2, 3) and c has shape (1, 6).  The 3 will not match the 6.